modules/account-lib/src/coin/algo/transaction.ts

```diff
@@ -99,17 +99,27 @@ export class Transaction extends BaseTransaction {
     const msigAddress = algosdk.multisigAddress(multiSigOptions);
     this._algoTransaction.from = algosdk.decodeAddress(msigAddress);
 
-    // Check if it is a signed or unsigned tx.
-    // If unsigned, sign it using first keypair and then append next signatures.
-    // If signed, appending next signatures.
-    let signedTx = this._signedTransaction
```

Comment:
What was wrong with this? Seems to be a undo from a merge conflict.

Reply:
I have changed this now. Please check